Makefile

```diff
@@ -1,55 +1,63 @@
-.PHONY : install agda repl libHtml testContainers test test-on-CI testHtml golden clean docs
+.PHONY : build install agda repl libHtml testContainers test succeed fail golden golden-succeed golden-fail clean docs fixWhitespace checkWhitespace
+
 FILES = $(shell find src -type f)
 
 build :
-	cabal build
+cabal build
```

Comment:
Here, and in all following goals, you deleted the TAB characters, but they are vital for `Makefile` syntax.  Please put them back.